eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -316,6 +316,9 @@ the main ServiceBusClientBuilder. -->
             files="com.azure.cosmos.ClientSideRequestStatistics"/> <!-- Need OperatingSystemMXBean from sun to obtain cpu info -->
   <suppress checks="EnforceFinalFields" files="com.azure.spring.cloud.config.AppConfigurationPropertySourceLocator"/>
   <suppress checks="ConstantName" files="com.azure.spring.cloud.config.AppConfigurationPropertySourceLocator"/>
+  <suppress checks="com.azure.tools.checkstyle.checks.GoodLoggingCheck"
+            files="[/\\]azure-cosmos-kafka-connect[/\\]"/>
+  <suppress checks="com.azure.tools.checkstyle.checks.ExternalDependencyExposedCheck" files="com.azure.cosmos.kafka.connect.CosmosDBSourceConnector"/>
```

Comment:
Disabling this one is bad (for the entire file) - would it be possible to use this pattern? Directly from the code - scoped to just the method violating the rule?

If not, this is fine...

https://github.com/Azure/azure-sdk-for-java/blob/38cf774a7accb8115c91ad2ce314db452f42daa2/sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/apachecommons/math/exception/util/LocalizedFormats.java#L44-L45

Reply:
will check

Reply:
will track in the next PR